### ISSUE TITLE
ADD CASCADIA CODE

### ISSUE THREAD

**Issue by Swastik-WhitehatJR** [google/fonts#3474]
ADD CASCADIA CODE

https://github.com/microsoft/cascadia-code

**Comment by DavidBruchmann** [comment]
correct me if I mistake, but I interpret the user name "
Swastik WhitehatJR"
as extremely racist.
I don't know how something like that is handled here.

**Comment by DavidBruchmann** [comment]
I think contributions with names like "Swastik-WhitehatJR" shouldn't be allowed as they are intended to transport a racist nazi-propaganda even also only in a subtle way which most font-users likely won't remark.
Therefore I vote this issue down without regard if it could be useful concerning the font itself.

**Comment by kenmcd** [comment]
> correct me if I mistake, but I interpret the user name "Swastik WhitehatJR" as extremely racist. I don't know how something like that is handled here.

Clueless knee-jerk reaction.
Swastik is the kids name.
WhitehatJR is the name of his school.

How should stupidity like that be handled here?

**Comment by DavidBruchmann** [comment]
@kenmcd  nevertheless the name has a message, especially in this combination of name parts.
I see neither fault or stupidity on my side.
The kid should consider changing it's real name if it's really "Swastika", even if in the Asian area swastika might not be related to nationalsozialism.

**Comment by kenmcd** [comment]
> @kenmcd nevertheless the name has a message, especially in this combination of name parts.
> I see neither fault or stupidity on my side.
> The kid should consider changing it's real name if it's really "Swastika", even if in the Asian area swastika might not be related to nationalsozialism.

So the entire world now needs to change their names because some moron decides misinterpret what it actually says to virtue signal.
The level of stupidity in that is stunning.

**Comment by tiroj** [comment]
स्वस्तिक is a Sanskrit word that means auspicious or lucky. It isn’t surprising to find it used as a personal name in India.

**Comment by moyogo** [comment]
@kenmcd don't insult others, no matter how mistaken they are.
While @DavidBruchmann is mistaken he did start with
> correct me if I mistake, but I interpret the user name "Swastik WhitehatJR" as extremely racist.
I don't know how something like that is handled here.

It is not something that can be nor should be handled here.
Swastik is a perfectly normal name where Swastik is from and has been long before “swastika” was misappropriated in Western culture.

**Comment by fitojb** [comment]
This issue is a duplicate of #2179

@Swastik-WhitehatJR, please search for existing issues before opening new ones.

**Comment by davelab6** [comment]
@Swastik-WhitehatJR thanks for posting your request to add that font. We are working on it, but need to get additional permissions from Microsoft before we can do so under that name. 

When I was a teenager, I remember posting similar sort of ALL CAPS requests online and not getting much response, and I benefited from reading http://www.catb.org/~esr/faqs/smart-questions.html :) Good luck with everything Sir
